### DISCUSSION TITLE
support for Apicur.io registry

### DISCUSSION THREAD

**Discussion by hussainsid** [AbsaOSS/ABRiS#362] [Q&A]
Hi , i am storing schemas on Apicur.io's registry, is it possible to use ABRiS library to fetch schema from Apicurio?

**Comment by kevinwallimann** [comment]
Hi @hussainsid 

This is not possible out of the box. A pull request to add support for Apicurio has been opened two years ago, but was unfortunately abandoned by the creator. See https://github.com/AbsaOSS/ABRiS/pull/301 
If you want to contribute this feature, feel free to create a new pull request
